example/test/01_test_web_login.ios.js

```diff
@@ -0,0 +1,103 @@
+
+const {
+  TWITTER_USER,
+  TWITTER_PASS,
+} = process.env
+
+const { driver, idFromXPath } = helper
+
+test('Twitter web login test', async (t) => {
+  const twitterButton = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeButton`
+  )
+  const accessAlert = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow[6]/XCUIElementTypeOther[2]/XCUIElementTypeAlert`
+  )
+  const agreeButton = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow[6]/XCUIElementTypeOther[2]/
+    XCUIElementTypeAlert/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther[3]/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[3]/
+    XCUIElementTypeButton`
+  )
+  const webEmailField = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther[3]/XCUIElementTypeOther`
+  )
+  const webPasswordField = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther[3]/XCUIElementTypeOther[2]`
+  )
+  const webAuthButton = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther[4]/XCUIElementTypeButton`
+  )
+  const loggedAlert = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther[2]/
+    XCUIElementTypeAlert/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeStaticText`
+  )
+  const loggedAlertOkButton = idFromXPath(`
+    /XCUIElementTypeApplication/XCUIElementTypeWindow/XCUIElementTypeOther[2]/
+    XCUIElementTypeAlert/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther[2]/
+    XCUIElementTypeOther[3]/XCUIElementTypeOther/XCUIElementTypeOther/XCUIElementTypeOther/
+    XCUIElementTypeButton`
+  )
+
+  try {
+    await driver.pause(240000).waitForVisible(twitterButton, 30000)
+    t.pass('User should see twitter login button')
+    await driver.click(twitterButton)
+    t.pass('User can click twitter button')
+  } catch (error) {
+    await helper.screenshot()
+    await helper.source()
+    throw error
+  }
```

Comment:
Wrap all test case logic in one try/catch like in other tests instead of many try/catch wrappers